README.md

```diff
@@ -1,8 +1,152 @@
+
+3. Quick Start:
+Below are commands to run to import the example data provided. See usage examples and documentation for more details.
+
+```
+(venv) > bin/cabin list
+(venv) > bin/cabin status
+(venv) > bin/cabin import --all --dry-run
+(venv) > bin/cabin import --all
+(venv) > bin/cabin status
+(venv) > bin/cabin shell
+```
+
+## Background
+This project came out of a startup where "cabin" was used for data collection and maintence. We sought to share some aspects of this tool we developed with the community. Carving out the tool from our code base resulted in some messiness that we tried to take care of but bare with us if artifacts are found here and there; please point them out and we would gladly sort out any confusion. We are aware of several areas/features that could use our attention, but again, it would be great to priortize based on needs or confusions that arise.
+
+## Usage examples
+*The following is an example, with detailed output, of how a dataset would be imported and updated*
+
+**See which datasets are available to work with, import all.** Note that:
+
+* the `import` command may specify a dataset directly or use `--all` to import all possible datasets that are not already imported
+* given the one example, `import --all` is effectively the same as specifying `import StormDetailsTable` because all dependancies of the table are verified either way
+* `import StormDetailsFile` however would only have triggered the download of the csv to the downloads directory
+* `--dry-run` allows a preview of which datasets will be affected
+
+```
+(venv) $ bin/cabin list
+StormDetailsFile
+StormDetailsOfficial
+StormDetailsTable
+(venv) cabin $ bin/cabin import --all
+[2022-01-13 14:07:47,492] INFO produce:  StormDetailsFile               with formula sha 1a1cd333 and root versions 2011
+downloads 100%[===================================================================>]  82.18M  5.86MB/s    in 15s     
+[2022-01-13 14:08:03,156] INFO Successfully downloaded to /Users/.../cabin/downloads/StormDetailsFile::2011::1a1cd333.csv
+[2022-01-13 14:08:03,158] INFO produced: StormDetailsFile               with formula sha 1a1cd333 and root versions 2011
+[2022-01-13 14:08:03,158] INFO produce:  StormDetailsTable              with formula sha ce78a5c3 and root versions 2011
+[2022-01-13 14:08:16,201] INFO Imported 79101 rows to table: StormDetailsTable::2011::ce78a5c3 
+[2022-01-13 14:08:16,261] INFO produced: StormDetailsTable              with formula sha ce78a5c3 and root versions 2011
+
+(venv) cabin $ bin/cabin status
+| version | table                             |   rows |    size | inputs               | outputs |
+|--------:|:----------------------------------|-------:|--------:|:---------------------|:--------|
+|    1  ✓ | StormDetailsTable::2011::ce78a5c3 | 79,101 | 19.4 MB | (1) StormDetailsFile | (0)     |
+```
```

Comment:
This section is quite verbose and a bit of a wall-of-text terminal dump.

Maybe the output of `import --all` can be skipped, just keeping the last line like so:
```
(venv) $ bin/cabin import --all
...
[2022-01-13 14:08:16,261] INFO produced: StormDetailsTable              with formula sha ce78a5c3 and root versions 2011
```